pygradle-plugin/src/test/groovy/com/linkedin/gradle/python/util/PackageInfoTest.groovy

```diff
@@ -22,56 +22,65 @@ class PackageInfoTest extends Specification {
 
     def 'can parse boring sdist'() {
         when:
-        def packageInfo = PackageInfo.fromPath('foo-1.0.0.tar.gz')
+        def packageInfo = PackageInfo.fromPath(new File('foo-1.0.0.tar.gz'))
         then:
         assert packageInfo.name == 'foo'
         assert packageInfo.version == '1.0.0'
     }
 
     def 'can parse snapshot sdist'() {
         when:
-        def packageInfo = PackageInfo.fromPath('foo-1.0.0-SNAPSHOT.tar.gz')
+        def packageInfo = PackageInfo.fromPath(new File('foo-1.0.0-SNAPSHOT.tar.gz'))
         then:
         assert packageInfo.name == 'foo'
         assert packageInfo.version == '1.0.0-SNAPSHOT'
     }
 
     def 'can parse a sdist that has "-" characters in the packageInfo.name'() {
         when:
-        def packageInfo = PackageInfo.fromPath('foo-bar-1.0.0.tar.gz')
+        def packageInfo = PackageInfo.fromPath(new File('foo-bar-1.0.0.tar.gz'))
         then:
         assert packageInfo.name == 'foo-bar'
         assert packageInfo.version == '1.0.0'
     }
 
     def 'can parse a sdist that has "-" characters in the packageInfo.version'() {
         when:
-        def packageInfo = PackageInfo.fromPath('foo-1.0.0-linkedin1.tar.gz')
+        def packageInfo = PackageInfo.fromPath(new File('foo-1.0.0-linkedin1.tar.gz'))
         then:
         assert packageInfo.name == 'foo'
         assert packageInfo.version == '1.0.0-linkedin1'
     }
 
     def 'can parse a sdist from a absolute path'() {
         when:
-        def packageInfo = PackageInfo.fromPath('/Users/sholsapp/.gradle/caches/modules-2/files-2.1/pypi/pex/0.8.5/5802dfe6dde45790e8a3e6f98f4f94219320f904/pex-0.8.5.tar.gz')
+        def packageInfo = PackageInfo.fromPath(new File('/Users/sholsapp/.gradle/caches/modules-2/files-2.1/pypi/pex/0.8.5/5802dfe6dde45790e8a3e6f98f4f94219320f904/pex-0.8.5.tar.gz'))
         then:
         assert packageInfo.name == 'pex'
         assert packageInfo.version == '0.8.5'
     }
 
     def 'can parse a windows path'() {
         when:
-        def packageInfo = PackageInfo.fromPath('Z:\\pygradle\\build\\ivy-repo\\pypi\\setuptools\\19.1.1\\setuptools-19.1.1')
+        def packageInfo = PackageInfo.fromPath(new File('Z:\\pygradle\\build\\ivy-repo\\pypi\\setuptools\\19.1.1\\setuptools-19.1.1'))
         then:
         assert packageInfo.name == 'setuptools'
         assert packageInfo.version == '19.1.1'
     }
 
     def 'can not parse a sdist that has an unknown extension'() {
         when:
-        PackageInfo.fromPath('foo-1.0.0.xxx')
+        PackageInfo.fromPath(new File('foo-1.0.0.xxx'))
```

Comment:
What do you think about replacing the `new File` with `makePackage` that returns a file, should in the future we want to make it actually create a file on disk?